include/vrv/timemap.h

```diff
@@ -15,6 +15,12 @@
 
 //----------------------------------------------------------------------------
 
+#include "horizontalaligner.h"
```

Comment:
I still find it a bit weird that `Fraction` is defined in `horizontalaligner`… I think I know why (fractions are mostly used for time, and time is usually on the horizontal) but it’s still a bit strange to be talking about “horizontal alignment” in a MIDI context.